WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -381,6 +401,57 @@ private extension DashboardViewModel {
     }
 }
 
+// MARK: InAppFeedback card
+//
+private extension DashboardViewModel {
+    /// Updates the card visibility state stored in `isInAppFeedbackCardVisible` by updating the app last feedback date.
+    ///
+    func onInAppFeedbackCardAction() {
+        let action = AppSettingsAction.updateFeedbackStatus(type: .general, status: .given(Date())) { [weak self] result in
+            guard let self = self else {
+                return
+            }
+
+            if let error = result.failure {
+                ServiceLocator.crashLogging.logError(error)
+            }
+
+            self.refreshIsInAppFeedbackCardVisibleValue()
+        }
+        stores.dispatch(action)
+    }
+
+    /// Calculates and updates the value of `isInAppFeedbackCardVisible`.
+    func refreshIsInAppFeedbackCardVisibleValue() {
+        let action = AppSettingsAction.loadFeedbackVisibility(type: .general) { [weak self] result in
+            guard let self = self else {
+                return
+            }
+
+            switch result {
+            case .success(let shouldBeVisible):
+                self.sendIsInAppFeedbackCardVisibleValueAndTrackIfNeeded(true)
```

Comment:
I think this part is accidentally changed to true.

Reply:
Great catch.  Fixed in https://github.com/woocommerce/woocommerce-ios/pull/12636/commits/49a320cb59986a0a8abf4e1787c2791d53c289b1